tanner/config.py

```diff
@@ -1,92 +1,44 @@
-import configparser
 import logging
 import os
 import sys
 
-LOGGER = logging.getLogger(__name__)
+import yaml
 
-config_template = {'DATA': {'db_config': '/opt/tanner/db/db_config.json',
-                            'dorks': '/opt/tanner/data/dorks.pickle',
-                            'user_dorks': '/opt/tanner/data/user_dorks.pickle',
-                            'crawler_stats': '/opt/tanner/data/crawler_user_agents.txt',
-                            'geo_db': '/opt/tanner/db/GeoLite2-City.mmdb',
-                            'tornado': '/opt/tanner/data/tornado.py',
-                            'mako': '/opt/tanner/data/mako.py'
-                            },
-                   'TANNER': {'host': '0.0.0.0', 'port': 8090},
-                   'WEB': {'host': '0.0.0.0', 'port': 8091},
-                   'API': {'host': '0.0.0.0', 'port': 8092, 'auth': False, 'auth_signature': 'tanner_api_auth'},
-                   'PHPOX': {'host': '0.0.0.0', 'port': 8088},
-                   'REDIS': {'host': 'localhost', 'port': 6379, 'poolsize': 80, 'timeout': 1},
-                   'EMULATORS': {'root_dir': '/opt/tanner'},
-                   'EMULATOR_ENABLED': {'sqli': True, 'rfi': True, 'lfi': True, 'xss': True, 'cmd_exec': True,
-                                        'php_code_injection': True, 'php_object_injection': True, "crlf": True,
-                                        "xxe_injection": True, "template_injection": True},
-                   'SQLI': {'type': 'SQLITE', 'db_name': 'tanner_db', 'host': 'localhost', 'user': 'root',
-                            'password': 'user_pass'},
-                   'XXE_INJECTION': {'OUT_OF_BAND': False},
-                   'RFI': {"allow_insecure": False},
-                   'DOCKER': {'host_image': 'busybox:latest'},
-                   'LOGGER': {'log_debug': '/opt/tanner/tanner.log', 'log_err': '/opt/tanner/tanner.err'},
-                   'MONGO': {'enabled': False, 'URI': 'mongodb://localhost'},
-                   'HPFEEDS': {'enabled': False, 'HOST': 'localhost', 'PORT': 10000, 'IDENT': '', 'SECRET': '',
-                               'CHANNEL': 'tanner.events'},
-                   'LOCALLOG': {'enabled': False, 'PATH': '/tmp/tanner_report.json'},
-                   'CLEANLOG': {'enabled': False},
-                   'REMOTE_DOCKERFILE': {'GITHUB': "https://raw.githubusercontent.com/mushorg/tanner/master/docker/"
-                                                   "tanner/template_injection/Dockerfile"},
-                   'SESSIONS': {"delete_timeout": 300}
-                   }
+LOGGER = logging.getLogger(__name__)
 
 
 class TannerConfig():
     config = None
 
+    @staticmethod
+    def read_config(path):
+        config_values = {}
+        with open(path, 'r') as f:
+            config_values = yaml.load(f, Loader=yaml.FullLoader)
```

Comment:
,I think it would be nice anyway to catch the exception instead of checking the ending of filename here https://github.com/mushorg/tanner/pull/371/files#diff-f428ee775d6702ef490b2120c5ef2ae5R23, because e.g. I've never used .ini for configs, I used `.config` :)